pkg/port/iscsit/iscsid.go

```diff
@@ -330,6 +330,9 @@ func (s *ISCSITargetDriver) iscsiExecLogin(conn *iscsiConnection) error {
 	conn.loginParam.isid = cmd.ISID
 	conn.loginParam.tsih = cmd.TSIH
 	conn.expCmdSN = cmd.CmdSN
+	conn.maxBurstLength = 262144
+	conn.maxRecvDataSegmentLength = 65536
```

Comment:
sorry for late, but could you please use a const variable instead of `262144` and `65536` ?